### PR TITLE
Make the travis script easier to read

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | This PR cleans the travis scripts, removes some useless or unused steps, fixes changes not applied etc.
| Type?         | improvement
| Category?     | TE
| BC breaks?    | Nope
| Deprecations? | Nope
| Fixed ticket? | /
| How to test?  | Tests must pass

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,58 +22,35 @@ php:
...
-  - sudo a2ensite prestashop.conf
-  - sudo service apache2 restart
+  # Apache & php-fpm configuration
+  - bash travis-scripts/setup-php-fpm.sh
+  - bash travis-scripts/setup-apache.sh
+  
+  # PrestaShop configuration
   - cp -Rf .composer/* ~/.composer/ & composer global install;
-  - wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
-  - sudo dpkg -i google-chrome*.deb
```
Comment (Quetzacoalt91) [inline .travis.yml]: Firefox is used with Selenium. We do not need to install Chrome anymore.

File: .travis.yml [hunk 2]
```diff
@@ -23,58 +22,35 @@ php:
...
-  - sudo a2enmod rewrite actions fastcgi alias
-  - echo "cgi.fix_pathinfo = 1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
-  - echo "memory_limit = 512M" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
-  - echo "always_populate_raw_post_data = -1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
-  - ~/.phpenv/versions/$(phpenv version-name)/sbin/php-fpm
-  - sudo cp -f tests/travis-ci-apache-vhost /etc/apache2/sites-available/prestashop.conf
-  - sudo cp -Rf . /var/www/prestashop.unit.test
-  - sudo chmod 777 -R /var/www/
-  - sudo a2dissite 000-default
-  - sudo a2ensite prestashop.conf
```
Comment (Quetzacoalt91) [inline .travis.yml]: Instead od copy pasting the whole folder content into `/var/www`, we set the Apache vhost to run with the Travis working directory.

File: .travis.yml [hunk 3]
```diff
@@ -23,58 +22,35 @@ php:
...
-  fast_finish: true
-
 before_install:
-  - if [[ ${TRAVIS_PHP_VERSION:0:3} == "7.0" ]]; then sudo cp tests/php7-pool.conf ~/.phpenv/versions/$(phpenv version-name)/etc/php-fpm.d/; fi
-  - phpenv config-rm xdebug.ini
-  - mysql -u root -e 'create database prestashop;'
-  - cp ~/.phpenv/versions/$(phpenv version-name)/etc/php-fpm.conf.default ~/.phpenv/versions/$(phpenv version-name)/etc/php-fpm.conf
-  - sudo a2enmod rewrite actions fastcgi alias
-  - echo "cgi.fix_pathinfo = 1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
-  - echo "memory_limit = 512M" >> ~/.phpenv/versions/$(phpenv version-name)/etc/php.ini
```
Comment (Quetzacoalt91) [inline .travis.yml]: This line was overwritten by travis.ini

File: .travis.yml [hunk 4]
```diff
@@ -23,58 +22,35 @@ php:
   - 5.6
   - 7.0
 
-matrix:
-  fast_finish: true
-
 before_install:
-  - if [[ ${TRAVIS_PHP_VERSION:0:3} == "7.0" ]]; then sudo cp tests/php7-pool.conf ~/.phpenv/versions/$(phpenv version-name)/etc/php-fpm.d/; fi
-  - phpenv config-rm xdebug.ini
-  - mysql -u root -e 'create database prestashop;'
```
Comment (Quetzacoalt91) [inline .travis.yml]: The PrestaShop installation can create the database for ourselves.

File: install-dev/index_cli.php [hunk 5]
```diff
@@ -31,7 +31,7 @@
 require_once dirname(__FILE__).'/init.php';
 require_once(__DIR__).DIRECTORY_SEPARATOR.'autoload.php';
 require_once _PS_INSTALL_PATH_.'classes/datas.php';
-ini_set('memory_limit', '128M');
+ini_set('memory_limit', '256M');
```
Comment (Quetzacoalt91) [inline install-dev/index_cli.php]: Required for parse error checks, using to much memory.

**Comment by aleeks** [comment]
Thank you @Quetzacoalt91

**Pull request merged**